core/src/lib.rs

```diff
@@ -291,6 +291,45 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    pub fn as_edn_value(&self) -> edn::Value {
+        let mut s = "[ ".to_string(); 
+        for (entid, attribute) in &self.schema_map {
+            s.push_str("{");
+            s.push_str(&format!(" :db/id :{:?}", entid));
```

Comment:
You're converting to an EDN string then parsing. That's probably not the right approach; the caller of this function is probably just going to turn it into a string again, perhaps first embedding it in another data structure!

Instead, you should build up an `edn::Value::Map`, which is simply a `BTreeMap` from keys to `edn::Value`s.